n_helpers.c

```diff
@@ -74,9 +79,204 @@ static short normalYearDaysByMonth[] = {0, 31, 59, 90, 120, 151, 181, 212, 243,
 static const char *daynames[] = {"Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"};
 
 // Forwards
+NOTE_C_STATIC void setTime(JTIME seconds);
 NOTE_C_STATIC bool timerExpiredSecs(uint32_t *timer, uint32_t periodSecs);
 NOTE_C_STATIC int ytodays(int year);
 
+//**************************************************************************/
+/*!
+  @brief  Receive a large binary object from the Notecard's binary buffer
+  @param  buffer A buffer to hold the result
+  @param  bufLen The total length of the buffer
+  @returns  NULL on success, else an error string pointer.
+  @note  Buffers are decoded in place, the original contents of the buffer will
+         be modified.
+*/
+/**************************************************************************/
+const char * NoteBinaryReceive(uint8_t * buffer, size_t bufLen)
```

Comment:
There's a discussion about this in the notecard-fw Slack channel. The Notecard can't do what you're describing today, but I (and Zak) want this feature, too.

Reply:
Unfortunately, that is not implemented on the Notecard. Both Hayden and I have independently requested this feature in the #notecard-fw channel.

Reply:
Is there an echo in here? :laughing:

Reply:
What can we do to make our case?

Reply:
Honestly, I don't think this is a priority item when considering all the other goings on. I'm certain it will have it's day, but I imagine it will be customer driven.

That being said, I would make mention of this in #notecard-fw, so there is another vote in the tally.